### ISSUE TITLE
create test messages

### ISSUE THREAD

**Issue by erwinspeybroeck** [adewg/ICAR#13]
dummy messages in github so they can be accessed

**Comment by ahokkonen** [comment]
example messages could be nested as part of "exampleUrlSchema" (inside each path's response or in components section) file or in separate files. I suggest we create separate example files for each event and add them as external component references into schema. With that approach will keep original schema clear.

**Comment by cookeac** [comment]
Yes, I like this approach. It also allows other organizations to define alternative url approaches for different purposes but reuse the examples. The example files themselves will assist with understanding so should be linked or referenced from any documentation.

**Comment by ahokkonen** [comment]
should examples be isolated form core files into own sub-folder - "examples" inside "Release Candidate Messages"? For each event there probably should be own example file aka. "IseminationEventExamples.json", "DryOffEventExamples"?

**Comment by cookeac** [comment]
Discussed in https://github.com/adewg/ICAR/wiki/Technical-Minutes-2020-01-31
@ahokkonen is contributing some example data for Finland.
